ChangeLog.md

```diff
@@ -160,15 +273,15 @@ We have finally released a new version of Haskell Language Server!
 Thanks for all contributors, many bugs has been fixed, and many features has landed.
 Here are the summary of changes:
 
-- Basic support for GHC 9.0.1 is added.  
+- Basic support for GHC 9.0.1 is added.
```

Comment:
These whitespaces should not be removed, as they are parts of formatting. (they means that the following text should be the separate line, not consecutive text.

Reply:
thanks for noting it, i have the editor configured to remove trailing whitespaces on save though, will turn off for this

Reply:
@Ailrun done!